spec/requests/body_weight_height_spec.rb

```diff
@@ -0,0 +1,86 @@
+require 'rails_helper'
+
+RSpec.describe "BodyWeightHeightController", type: :request do
```

Comment:
nice to have kalo dipindah ke folder `spec/controller` dana namanya ngikut nama file controller-nya dikasih suffix `_spec`